network/network_windows.go

```diff
@@ -290,10 +290,14 @@ func (nm *networkManager) configureHcnNetwork(nwInfo *NetworkInfo, extIf *extern
 		hcnNetwork.Type = hcn.L2Bridge
 	case opModeTunnel:
 		hcnNetwork.Type = hcn.L2Tunnel
+	case opModeTransparent:
+		hcnNetwork.Type = hcn.Transparent
+		hcnNetwork.Flags = 1024
```

Comment:
todo: determine later if we need this flag constant or hns changes will support this